modules/primer-breadcrumb/lib/breadcrumb.scss

```diff
@@ -1,6 +1,5 @@
 .breadcrumb-item {
   display: inline-block;
-  margin-left: -$spacer-1;
```

Comment:
We don't want the margin left on the first breadcrumb,  but we need it on the others, this change makes the spacing either side of the `/` uneven.